From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Thu, 2 Aug 2018 13:07:40 +0200
Subject: latex syntax

Need curly braces on arg to \begin, need arg to \right
---
 src/mlpack/methods/ann/layer/flexible_relu.hpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/mlpack/methods/ann/layer/flexible_relu.hpp b/src/mlpack/methods/ann/layer/flexible_relu.hpp
index a9c837e..3117d7d 100644
--- a/src/mlpack/methods/ann/layer/flexible_relu.hpp
+++ b/src/mlpack/methods/ann/layer/flexible_relu.hpp
@@ -27,11 +27,11 @@ namespace ann /**Artificial Neural Network*/ {
  * @f{eqnarray*}{
  * f(x) &=& \max(0,x)+alpha \\
  * f'(x) &=& \left\{
- * 	 \begin(array){lr}
+ * 	 \begin{array}{lr}
  * 	   1 & : x > 0 \\
  * 	   0 & : x \le 0
  * 	 \end{array}
- * \right
+ * \right.
  * @f}
  *
  * For more information, read the following paper:
